src/_locales/fi/messages.json

```diff
@@ -0,0 +1,495 @@
+    "description": "Option of error correction level for QR code. See https://en.wikipedia.org/wiki/QR_code#Error_correction.",
+    "hash": "0ea96235848be6f66788cad72bc0ea02"
+  },
+  "optionEcQuartile": {
+    "message": "Neljännes (25%)",
+    "description": "Option of error correction level for QR code. See https://en.wikipedia.org/wiki/QR_code#Error_correction.",
+    "hash": "344e853acdaef0ac95e161cdbc201036"
+  },
+  "optionEcHigh": {
+    "message": "Korkea (25%)",
+    "description": "Option of error correction level for QR code. See https://en.wikipedia.org/wiki/QR_code#Error_correction.",
+    "hash": "30035374d52e135473d464fe1726ad8c"
+  },
+  "optionErrorCorrectionDescr": {
+    "message": "Tämä arvo tarkoittaa miten suuri osa koodista voi puuttua, jotta se voidaan silti skannata onnistuneesti.",
+    "description": "The description of the error correction option.",
+    "hash": "2bbbd8aafc044a910d26cbf04e560766"
+  },
+  "optionErrorCorrectionDescrLink": {
+    "message": "https://en.wikipedia.org/wiki/QR_code#Error_correction",
+    "description": "The 'Learn more' link in the description of the error correction option.",
+    "hash": "530f6b1ee177dddd7d40571d98a64408"
+  },
+  "optionAutoGetSelectedText": {
+    "message": "Käytä automaattisesti nettisivulta valittua tekstiä",
+    "description": "This is an option shown in the add-on settings.",
+    "hash": "21667c5a30f89561cea02351172b9c91"
+  },
+  "optionUseMonospaceFont": {
+    "message": "Monospace-fontti",
+    "description": "This is an option shown in the add-on settings.",
+    "hash": "69191273c436064be8c4f5d6a8f675f5"
+  },
+  "optionDebugMode": {
+    "message": "Aktivoi debug-tila",
+    "description": "This is an option shown in the add-on settings.",
+    "hash": "de6506af4973e0980d64f242fba11a19"
+  },
+  "optionDebugModeDescr": {
+    "message": "Tämä on hyödyllinen vain saadaksesi lisätietoa bugien raportoimista ym. varten.",
+    "description": "The description shown for the debug mode (optionDebugMode).",
+    "hash": "2aff10770796694827ec4253860e0771"
+  },
+  "translatorCredit": {
+    "message": "Tämä selainlaajennus on käännetty englanniksi seuraavien kääntäjien toimesta: $TRANSLATORS$.",
+    "description": "The credit text for the translator. See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translator-credit-inside-of-add-on for how to translate this.",
+    "placeholders": {
+      "translators": { "content": "$1", "example": "<a href=\"https://github.com/rugk/\">@rugk</a>" }
+    },
+    "hash": "a6f5deecd624d5f8cd58350c72dd8fdf"
+  },
+  "translatorLink": {
+    "message": "https://github.com/rugk",
+    "description": "The link to the translator's GitHub profile.",
+    "hash": "4775fb2f88fd33d02b871e5021dec3e9"
+  },
+  "translatorUsername": {
+    "message": "rugk",
+    "description": "The username that the translator wants to be referred to.",
+    "hash": "ce96c18a8a1b4b528b294e981ee4d651"
+  },
```

Comment:
To honor your work, you should be credited in the extension. To do that, you need to provide your name and a link in the extension.

You can credit yourself here and provide a link (in `translatorLink`) and a name (in `translatorUsername`). Also, you should add yourself to the CONTRIBUTORS.md file and the changelog as a translator to [give credits here](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#credit).

See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translations for how these placeholders are translated, especially [the official doc](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Internationalization#Providing_localized_strings_in__locales).